packages/devtools_server/lib/src/external_handlers.dart

```diff
@@ -34,11 +34,14 @@ Future<shelf.Handler> defaultHandler(
 
   // Default static handler for all non-package requests.
   Handler? buildDirHandler;
+  File? indexFile;
```

Comment:
not your fault, but can you remove line 13 'import 'package:shelf/shelf.dart';' as that is a duplicate import